addons/account/models/account_move.py

```diff
@@ -1267,7 +1267,8 @@ def _compute_partner_credit_warning(self):
                            move.move_type == 'out_invoice' and \
                            move.company_id.account_use_credit_limit
             if show_warning:
-                updated_credit = move.partner_id.credit + move.amount_total_signed
+                amount_total_currency = move.tax_totals['amount_total'] / move.currency_id.rate
```

Comment:
Use `move.invoice_line_ids[0].currency_rate` instead to get the rate at the right date
`currency_id.rate` will always get the latest rate, which might not be the one used by the invoice.

Reply:
I used `move.currency_id._convert()` instead, otherwise an additional check was necessary to make sure there are `invoice_line_ids` (invoice creation), which would have added more code anyway :wink: